apps/builder/app/canvas/inspector-edits.ts

```diff
@@ -0,0 +1,12 @@
+import { $inspectorLastInputTime, $props } from "~/shared/nano-states";
+import { setInert } from "./shared/inert";
+
+export const subscribeInspectorEdits = () => {
+  const unsubscribeInputTime = $inspectorLastInputTime.listen(setInert);
```

Comment:
why do we need a date vs counter? just a thought.

Reply:
or even just a true/false switch, seems like you are just using it as a way to trigger set setInert, so could be basically anything, date made me think its somehow depending on the date value

Reply:
For me Date.now() is just simpliest form as there is no need to read state etc.
And with Date easier to give a name to variable. 
Have no strong opinion here, lastInputTime has some physical sense as it says that actions depending on it somehow depends on time (and they are), counter or switch have not.

Reply:
IMO the main issue in cases like this are using atoms and subscirptions instead of imperative calls into a canvas.

How it would be better to call `canvas.setInert()` instead of 3 atoms and 3 subscriptions we have to call setInert.